core/workspace_svg.js

```diff
@@ -749,7 +750,9 @@ Blockly.WorkspaceSvg.prototype.createDom = function(opt_backgroundClass) {
     if (!Blockly.Toolbox) {
       throw Error('Missing require for Blockly.Toolbox');
     }
-    this.toolbox_ = new Blockly.Toolbox(this);
+    var toolboxClass = Blockly.registry.getClassFromOptions(
```

Comment:
Since this is a constructor we're returning, the style guide suggests its name be in `UpperCamelCase`. so this should be `var ToolboxClass`

Reply:
done